app/scripts/modules/core/src/managed/constraints/ManualJudgement.tsx

```diff
@@ -21,7 +21,8 @@ export const ManualJudgementTitle = ({ constraint }: { constraint: IConstraint }
     case 'FAIL':
       return (
         <span>
-          Something went wrong <span className="text-regular">—</span>{' '}
+          {constraint.comment?.includes('Timed out') ? 'Manual judgement was rejected' : 'Manual judgement failed'}{' '}
+          <span className="text-regular">—</span>{' '}
```

Comment:
This seems pretty brittle, but I'm guessing theres' nothing better to key off of?  Would it be possible to flip the detection logic such that the `Manual judgement was rejected` case is explicitly checked for and everything else falls back to `Manual judgement failed` message?